doc/source/getting_started/install.rst

```diff
@@ -62,8 +62,8 @@ then use the following command from within this local directory:
 
    pip install --no-index --find-links=. ansys-dpf-core
 
-Beware that PyDPF-Core wheelhouses do not include the optional plotting dependencies.
-To allow for plotting capabilities, also download the wheels corresponding to your platform and Python interpreter version
+Note that PyDPF-Core wheelhouses do not include the optional plotting dependencies.
+To use the plotting capabilities, also download the wheels corresponding to your platform and Python interpreter version
 for `PyVista <https://pypi.org/project/pyvista/#files>`_ and 
 `matplotlib <https://pypi.org/project/matplotlib/#files>`_, then place them in the same previous local directory and run the command above.
```

Comment:
```suggestion
`matplotlib <https://pypi.org/project/matplotlib/#files>`_. Then, place them in the same local directory and run the preceding command.
```
Google dev style guidelines say to avoid location indicators like above and below.